### PR TITLE
fotografde/cakephp-sms

### PR DESCRIPTION
**Pull request by felixmaier1989**
We've developed a CakePHP [plugin](https://github.com/fotografde/cakephp-sms) to send SMS out. We basically copied the code for CakeEmail and adjusted it to telephone short messaging requirements.

### CODE REVIEW

**Comment by dereuromark** [comment]
Any update on my initial question? Why this justifies an own subpoint? Could just go into the linked section @ https://github.com/FriendsOfCake/awesome-cakephp/pull/98

File: README.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ Focus is on the specific task.
 	- [Debugging](#debugging)
 	- [Dependency Injection](#dependency-injection)
 	- [E-commerce](#e-commerce)
+	- [Email](#email)
```
Comment (dereuromark) [inline README.md]: thx! :)

**Comment by dereuromark** [comment]
I am not totally convinced of the "telephone calls" part ;) But unless someone else has a better sentence, merging.

File: README.md [hunk 2]
```diff
@@ -108,6 +110,12 @@ Focus is on the specific task.
 - [Postmark plugin](https://github.com/maurymmarques/postmark-cakephp) - Makes email delivery using Postmark.
 - [SendGrid plugin](https://github.com/a2design-company/sendgrid-webapi-cakephp-plugin) - CakePHP plugin for SendGrid WebAPI.
 
+## Telephone
+*Plugins for telephone calls, messaging.*
+
+- [SMSFly plugin](https://github.com/imsamurai/cakephp-sms-fly-datasource) - Data source for sending SMS via sms-fly.
+- [Sms plugin](https://github.com/fotografde/cakephp-sms) - Send SMS as easily as with CakeEmail.
```
Comment (jadb) [inline README.md]: What's CakeEmail? And shouldn't it be 'SMS' (all caps)?

Comment (jadb) [inline README.md]: Also, SMS should come before SMSFly (in alphabetical order).

**Comment by jadb** [comment]
Not keen on 'Telephone'. How about 'Transports'? And how about just making 'Transports (email, sms, etc.)'?

File: README.md (hunk 2 above)
Comment (dereuromark) [inline README.md]: CakeEmail is the 2.x email wrapper ;)
SMS as class name? Definitely not :P

**Comment by dereuromark** [comment]
We could probably summarize those as Transports indeed.

File: README.md (hunk 2 above)
Comment (jadb) [inline README.md]: Arg!! Sorry, hadn't realized it's a 2.0 PR ;)

**Pull request merged**